src/components/shared/Popup.js

```diff
@@ -70,32 +68,31 @@ export default function PopupBox(props) {
     return null
   }
 
+  console.log(demo1)
```

Comment:
remove `console.log`

Reply:
🤦 Cypress was failing, wonder if we can upddate eslint to pick up on console.logs

Reply:
it has been removed